packages/mds-test-data/index.ts

```diff
@@ -552,6 +553,9 @@ function makeTrip(device: Device): Trip {
   }
 }
 
+const SCOPED_AUTH = (scopes: AccessTokenScope[], provider_id = TEST1_PROVIDER_ID) =>
```

Comment:
Since this is not _always_ a `provider_id`, perhaps call this argument `principalId` to match how it is parsed in `mds-api-authorizer`

Reply:
👍